backend/cypress/e2e/audit-findings-text.cy.js

```diff
@@ -1,36 +1,40 @@
 
 describe('Audit findings text page', () => {
-    before(() => {
-      cy.visit('/report_submission/audit-findings-text/2022XB40001000002');
-    });
-    it('Page loads successfully', () => {
-      cy.url().should('include','/report_submission/audit-findings-text/2022XB40001000002');
-    });
+  const reportTestId = '2023MAY0001000001'
 
+  before(() => {
+    cy.visit(`/report_submission/audit-findings-text/${reportTestId}`);
+  });
+  it('Page loads successfully', () => {
+    cy.url().should('include', `/report_submission/audit-findings-text/${reportTestId}`);
   });
 
-  describe('File upload successful', () => {
+  describe('findings text workbook upload successful', () => {
     it('Successfully uploads audit findings text', () => {
-      cy.intercept('/audit/excel/FindingsText/*', {
-        fixture: 'success-res.json', }).as('uploadSuccess')
-      cy.visit('report_submission/audit-findings-text/2022XB40001000002');
-      cy.get('#file-input-audit-findings-text-xlsx').attachFile('FindingsText2019-2022.xlsx');
+      cy.intercept('/audit/excel/findings-text/*', {
+        fixture: 'success-res.json',
+      }).as('uploadSuccess')
+      cy.visit(`/report_submission/audit-findings-text/${reportTestId}`);
+      cy.get('#file-input-audit-findings-text-xlsx').attachFile('findings-text-UPDATE.xlsx');
       cy.wait('@uploadSuccess').its('response.statusCode').should('eq', 200)
-      cy.wait(2000).get('#info_box').should('have.text','File successfully validated!');
+      cy.wait(2000).get('#info_box').should('have.text', 'File successfully validated! Your work has been saved.');
       cy.get('#continue').click();
+      cy.url().should('contain', `/audit/submission-progress/${reportTestId}`);
     })
+  });
 
-    describe('File upload fail', () => {
-      it('unsuccessful upload audit findings text', () => {
-        cy.intercept('POST','/audit/excel/FindingsText/*', {
-          statusCode: 400,
-          fixture: 'fail-res.json', }).as('uploadFail')
-        cy.visit('report_submission/audit-findings-text/2022XB40001000002');
-        cy.get('#file-input-audit-findings-text-xlsx').attachFile('FederalAwardsExpendedTemplateUG2019.xlsx');
-        cy.wait('@uploadFail').its('response.statusCode').should('eq', 400)
-        cy.wait(2000).get('#info_box').should('have.text','Error on validation. See console for more information.');
-      })
+  describe('File upload fail', () => {
+    it('unsuccessful upload audit findings text', () => {
+      cy.intercept('POST', '/audit/excel/findings-text/*', {
+        statusCode: 400,
+        fixture: 'fail-res.json',
+      }).as('uploadFail')
+      cy.visit(`/report_submission/audit-findings-text/${reportTestId}`);
+      cy.get('#file-input-audit-findings-text-xlsx').attachFile('federal-awards-Test.xlsx');
+      cy.wait('@uploadFail').its('response.statusCode').should('eq', 400)
+      cy.wait(2000).get('#info_box').should('have.text', 'Field Error: undefined');
```

Comment:
**question**: I see where you are using this text to be able to handle the 400 result using the `#info_box` element, but I'm not sure if the frontend team is ready to make the change that uncaught upload errors should show up on the page like this. @jperson1 @heymatthenry Can you look at this and the change in `static/js/upload-page.js`?

Reply:
As of right now, that info box will populate with a number of different things depending on what's uploaded. The change in `static/js/upload-page.js` is a catch all for an unexpected field error (this suggests an incorrect template). That's fine for now, because on `main` the user may receive no feedback on the page. The error handling in there is due for some changes, some of which are captured in this [draft PR](https://github.com/GSA-TTS/FAC/pull/1517). If this goes in before I complete that PR, I'll be sure to update the tests accordingly. 

TL;DR I think it's fine, and it will certainly be fine after https://github.com/GSA-TTS/FAC/pull/1517 and more copy comes in from the design folks.